examples/docs/en-US/alert.md

```diff
@@ -233,6 +233,7 @@ Description includes a message with more detailed information.
 
 | Name | Description |
 |------|--------|
+| — | description |
```

Comment:
`| description | ... |`

Reply:
`<p class="el-alert__description" v-if="$slots.default && !description"><slot></slot></p>`

I saw a similar code where the default slot is "—" and this slot is used for description.

So.. What needs to be corrected?

Reply:
my fault.